.travis.yml

```diff
@@ -98,6 +88,13 @@ install:
 # Let's enable rabbitmqadmin
 # See https://github.com/messagebus/lapine/wiki/Testing-on-Travis.
 before_script:
+  # key_url no longer works for APT addon
+  # 1. Install MongoDB 3.4 and latest version of git
+  - sudo add-apt-repository ppa:git-core/ppa
+  - curl https://www.mongodb.org/static/pgp/server-3.4.asc | sudo apt-key add -
+  - echo "deb [arch=amd64] http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.4 multiverse" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update -qq
+  - sudo apt-get install mongodb-org-server mongodb-org-shell git -y
```

Comment:
Ideally to move this section at early point before-install where also another apt-get update happens